astropy/table/tests/test_table.py

```diff
@@ -331,6 +331,29 @@ def test_reverse_mixin(self):
         assert np.allclose(t['sc'].ra.to_value('deg'), [2, 1])
 
 
+@pytest.mark.usefixtures('table_types')
+class TestRound():
+
+    def test_round_int(self, table_types):
+        t = table_types.Table([['a', 'b', 'c'],
+                               [1.11, 2.3, 3.0],
+                               [1.123456, 2.9876, 3.901]])
+        t.round()
+        assert np.all(t['col0'] == np.array(['a', 'b', 'c']))
```

Comment:
The `np.array(...)` on the right side values is not necessary.  The code is a bit easier to read without it.